addons/web/static/src/js/views/kanban/kanban_column.js

```diff
@@ -141,7 +142,7 @@ var KanbanColumn = Widget.extend({
         }
         var self = this;
         var width = this.records.length ? this.records[0].$el.innerWidth() : this.$el.width() - 8;
-        this.quickCreateWidget = new RecordQuickCreate(this, width);
+        this.quickCreateWidget = new RecordQuickCreate(this, width, {default_name: this.data.getContext().default_name});
```

Comment:
Isn't it going to crash if there is no 'default_name' ?
I think we should have @ged-odoo advice here.